src/fv/mpfao/tdympfao.c

```diff
@@ -794,6 +862,15 @@ PetscErrorCode TDySetDMFields_TH_MPFAO(void *context, DM dm) {
   PetscFunctionReturn(0);
 }
 
+PetscErrorCode TDySetDMFields_Salinity_MPFAO(void *context, DM dm) {
+  PetscFunctionBegin;
+  PetscErrorCode ierr;
+  // Set up the section, 2 dofs per cell.
+  ierr = SetFields(dm, 2, (const char*[2]){"LiquidPressure", "Concentration"},
```

Comment:
`Concentration` --> `SalineConcentration`

Reply:
I should also change `SetBoundarySalinity` (and maybe the corresponding source/sink term) to match whatever term we select.